code/modules/research/designs/guncrafting_designs.dm

```diff
@@ -0,0 +1,544 @@
+
+// magazines
+/datum/design/mag_ar
+	name = "Gun Magazine: Assault Rifle"
+	desc = "see title"
+	id = "mag_ar"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 4000)
+	build_path = /obj/item/ammo_box/magazine/guncrafting_ar
+	category = list("Gun Ammo")
+
+/datum/design/mag_pistol
+	name = "Gun Magazine: Pistol"
+	desc = "see title"
+	id = "mag_pistol"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 8000)
+	build_path = /obj/item/ammo_box/magazine/guncrafting_pistol
+	category = list("Gun Ammo")
+
+/datum/design/mag_357
+	name = "Gun Magazine: .357 Speedloader"
+	desc = "see title"
+	id = "mag_357"
+	req_tech = list("combat" = 1, "materials" = 1, "syndicate" = 2)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 10000) // Cheaper than the autolathe counterpart, which the cost of that is 30,000
+	build_path = /obj/item/ammo_box/a357
+	category = list("Gun Ammo")
+
+/datum/design/mag_38
+	name = "Gun Magazine: .38 Speedloader"
+	desc = "see title"
+	id = "mag_38"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 4000)
+	build_path = /obj/item/ammo_box/c38
+	category = list("Gun Ammo")
+
+/datum/design/slugshell
+	name = "Shotgun Shell: Slug"
+	desc = "A slug shell."
+	id = "slugshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun
+	category = list("Gun Ammo")
+
+/datum/design/buckshotshell
+	name = "Shotgun Shell: Buckshot"
+	desc = "A buckshot shell."
+	id = "buckshotshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/buckshot
+	category = list("Gun Ammo")
+
+/datum/design/rubbershotshell
+	name = "Shotgun Shell: Rubber Shot"
+	desc = "A rubber shot shell."
+	id = "rubbershotshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/rubbershot
+	category = list("Gun Ammo")
+
+/datum/design/beanbagslugshell
+	name = "Shotgun Shell: Beanbag Slug"
+	desc = "A beanbag slug shell."
+	id = "beanbagslugshell"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/beanbag
+	category = list("Gun Ammo")
+
+/datum/design/dartshell
+	name = "Shotgun Shell: Dart"
+	desc = "A dart shell."
+	id = "dartshell"
+	req_tech = list("combat" = 1, "materials" = 3)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 1000)
+	build_path = /obj/item/ammo_casing/shotgun/dart
+	category = list("Gun Ammo")
+
+/datum/design/dbshell
+	name = "Shotgun Shell: Dragon's Breath"
+	desc = "A dragon's breath shell."
+	id = "dbshell"
+	req_tech = list("combat" = 3, "materials" = 3)
```

Comment:
add plasma = 4 here.

Also, add 1000 metal and 1000 plasma to its required materials.